src/components/organisms/ExplorerPane/FilePane/FileSystemTree/TreeNode/hooks.tsx

```diff
@@ -552,6 +562,24 @@ export const useFolderMenuItems = (
       },
     });
 
+    newMenuItems.push({
+      disabled: isRoot || (!fileEntry.children && !fileEntry.isSupported),
+      key: 'update_scanning',
+      label: `${
+        fileEntry.isExcluded && projectConfig?.scanExcludes?.some(e => micromatch.isMatch(filePath, e))
+          ? 'Remove from'
+          : 'Add to'
+      } Files: Exclude`,
+      onClick: (e: any) => {
+        e.domEvent.stopPropagation();
+        if (fileEntry.isExcluded) {
+          removeEntryFromScanExcludes(filePath);
```

Comment:
will these automatically trigger a rescan?

Reply:
We currently have a popup warning about the reload 
![image](https://github.com/kubeshop/monokle/assets/47887589/effbbdc6-6936-496c-9203-51504f05b972)